### ISSUE TITLE
This plugin is not production of JetBrains

### ISSUE THREAD

**Issue by Vlados177** [getgauge/Intellij-Plugin#414]
### Expected behavior
Tell us what should happen

### Actual behavior
Tell us what happens instead

### Steps to reproduce
1.
2.
3.

### Version information

* IntelliJ version and edition(Community/Ultimate): 
* OS information:
* Gauge IntelliJ plugin version:
* Does the project uses any Build tool(Gradle/Maven)?
```
Run gauge -v on your system and paste the results here
```

**Comment by Vlados177** [comment]
java.lang.reflect.InvocationTargetException
	at jdk.internal.reflect.GeneratedMethodAccessor66.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.intellij.execution.process.ProcessHandler$2.invoke(ProcessHandler.java:214)
	at com.sun.proxy.$Proxy26.onTextAvailable(Unknown Source)
	at com.intellij.execution.process.ProcessHandler.notifyTextAvailable(ProcessHandler.java:188)
	at com.intellij.execution.process.ColoredProcessHandler.textAvailable(ColoredProcessHandler.java:83)
	at com.intellij.execution.process.ColoredProcessHandler.coloredTextAvailable(ColoredProcessHandler.java:57)
	at com.intellij.execution.process.AnsiEscapeDecoder.processTextChunk(AnsiEscapeDecoder.java:99)
	at com.intellij.execution.process.AnsiEscapeDecoder.escapeText(AnsiEscapeDecoder.java:63)
	at com.intellij.execution.process.ColoredProcessHandler.notifyTextAvailable(ColoredProcessHandler.java:46)
	at com.intellij.execution.process.BaseOSProcessHandler$SimpleOutputReader.onTextAvailable(BaseOSProcessHandler.java:179)
	at com.intellij.util.io.BaseOutputReader.sendText(BaseOutputReader.java:211)
	at com.intellij.util.io.BaseOutputReader.processInput(BaseOutputReader.java:195)
	at com.intellij.util.io.BaseOutputReader.readAvailableNonBlocking(BaseOutputReader.java:114)
	at com.intellij.util.io.BaseDataReader.readAvailable(BaseDataReader.java:77)
	at com.intellij.util.io.BaseDataReader.doRun(BaseDataReader.java:155)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:208)
	at com.intellij.util.io.BaseDataReader.lambda$start$0(BaseDataReader.java:61)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Unterminated string at line 2 column 1 path $.message
	at com.google.gson.Gson.fromJson(Gson.java:942)
	at com.google.gson.Gson.fromJson(Gson.java:892)
	at com.google.gson.Gson.fromJson(Gson.java:841)
	at com.google.gson.Gson.fromJson(Gson.java:813)
	at com.thoughtworks.gauge.execution.runner.GaugeOutputToGeneralTestEventsProcessor.processServiceMessages(GaugeOutputToGeneralTestEventsProcessor.java:61)
	at com.intellij.execution.testframework.sm.runner.OutputToGeneralTestEventsConverter.processConsistentText(OutputToGeneralTestEventsConverter.java:114)
	at com.intellij.execution.testframework.sm.runner.OutputToGeneralTestEventsConverter$1.onTextAvailable(OutputToGeneralTestEventsConverter.java:71)
	at com.intellij.execution.testframework.sm.runner.OutputEventSplitter.flushInternal(OutputEventSplitter.kt:166)
	at com.intellij.execution.testframework.sm.runner.OutputEventSplitter.flushInternal$default(OutputEventSplitter.kt:151)
	at com.intellij.execution.testframework.sm.runner.OutputEventSplitter.processInternal(OutputEventSplitter.kt:98)
	at com.intellij.execution.testframework.sm.runner.OutputEventSplitter.process(OutputEventSplitter.kt:72)
	at com.intellij.execution.testframework.sm.runner.OutputToGeneralTestEventsConverter.process(OutputToGeneralTestEventsConverter.java:97)
	at com.intellij.execution.testframework.sm.SMTestRunnerConnectionUtil$2.onTextAvailable(SMTestRunnerConnectionUtil.java:213)
	... 24 more
Caused by: com.google.gson.stream.MalformedJsonException: Unterminated string at line 2 column 1 path $.message
	at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1568)
	at com.google.gson.stream.JsonReader.nextQuotedValue(JsonReader.java:1031)
	at com.google.gson.stream.JsonReader.nextString(JsonReader.java:815)
	at com.google.gson.internal.bind.TypeAdapters$16.read(TypeAdapters.java:402)
	at com.google.gson.internal.bind.TypeAdapters$16.read(TypeAdapters.java:390)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:131)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:222)
	at com.google.gson.Gson.fromJson(Gson.java:927)
	... 36 more

**Comment by negiDharmendra** [comment]
- run `gauge -v` on your terminal and share the output with us
- which os your working on
- java version
- Intellij version and Gauge Intellij Plugin version.
- Were you able to replicate this issue with a new Gauge Java project. If not is it possible for you to share the sample project.

**Comment by Vlados177** [comment]
csharp (0.10.6)
dotnet (0.1.2)
flash (0.0.1)
html-report (4.0.8)
java (0.7.3)
python (0.3.6)
ruby (0.5.2)
screenshot (0.0.1)
spectacle (0.1.3)
xml-report (0.2.2)
---
Windows 10
-----------
java 11
---------
IntelliJ IDEA 2019.2.3 x64 and Gauge 0.3.18
-----------------
Cannot share the project

**Comment by Vlados177** [comment]
![image](https://user-images.githubusercontent.com/58368321/70393712-8b1f7d00-19f5-11ea-9376-5d2c09cb4fa1.png)
Please, help with problem it!

**Comment by Vlados177** [comment]
I resolved this problem, only after installed java 8
![image](https://user-images.githubusercontent.com/58368321/70434576-41827100-1a8d-11ea-879e-60e66416405e.png)

**Comment by nehashri** [comment]
duplicate of https://github.com/getgauge/gauge/issues/1528
